tests/test_enhanced_operators_real_api.py

```
@@ -0,0 +1,264 @@
+    
+    high_scoring_idea = GeneratedIdea(
+        content="AI-powered smart waste sorting system with IoT sensors",
+        thinking_method=ThinkingMethod.ABDUCTION,
+        agent_name="TestAgent",
+        generation_prompt="Advanced waste management",
+        confidence_score=0.9,
+        reasoning="High-tech solution with proven effectiveness",
+        metadata={
+            "generation": 2,
+            "overall_fitness": 0.88,  # High fitness - should trigger breakthrough
+            "overall_fitness": 0.85
```

Ruff (0.14.5)

Dictionary key literal `"overall_fitness"` repeated

(F601)